### PR TITLE
add toast & make tooltip cross-platform

### PR DESCRIPTION
**Pull request by teimurjan**
# What's done?

1. Added a toast component to use in the notifications.
2. The lib didn't work correctly on mobile because `ToolTip` component was using MUI (a lib non-compatible with RN). That's why I migrated `Tooltip` to `native-base` which is cross-platform lib.
3. Secured font HTML element creation by checking if `document` exists.
4. Used `svgr` loader with `native: true` in order to make SVGs available in both web & mobile.

# Screenshots

## Toasts

|Web|Mobile|
|-|-|
|Screen Shot 2022-09-08 at 12 11 19|Screen Shot 2022-09-08 at 12 13 07|

### Light
Screen Shot 2022-09-07 at 12 49 32

### Dark
Screen Shot 2022-09-07 at 12 49 40

### CODE REVIEW

File: packages/design-system/package.json [hunk 1]
```diff
@@ -8,10 +8,7 @@
     "dist"
   ],
   "peerDependencies": {
-    "@emotion/react": "^11.7.1",
-    "@emotion/styled": "^11.6.0",
```
Comment (teimurjan) [inline packages/design-system/package.json:12]: Not used

File: packages/design-system/package.json [hunk 2]
```diff
@@ -8,10 +8,7 @@
...
     "@ethersproject/providers": "^5.4.5",
-    "@mui/material": "^5.4.1",
```
Comment (teimurjan) [inline packages/design-system/package.json:14]: Not used anymore

File: packages/design-system/package.json [hunk 3]
```diff
@@ -52,7 +49,6 @@
     "@storybook/addon-essentials": "^6.4.18",
     "@storybook/addon-links": "^6.4.18",
     "@storybook/addon-postcss": "^2.0.0",
-    "@storybook/addon-react-native-web": "^0.0.18",
```
Comment (teimurjan) [inline packages/design-system/package.json:55]: Not used

File: packages/design-system/package.json [hunk 4]
```diff
@@ -71,9 +67,10 @@
     "customize-cra": "^1.0.0",
     "postcss": "8",
     "react": "^17.0.2",
-    "react-app-rewired": "^2.2.1",
```
Comment (teimurjan) [inline packages/design-system/package.json:74]: Not used

File: packages/design-system/rollup.config.js [hunk 5]
```diff
@@ -46,7 +46,7 @@ const getConfig = (format) => ({
       include: ['**/*.ttf'],
       limit: Infinity,
     }),
-    svgr(),
+    svgr({ native: true }),
```
Comment (teimurjan) [inline packages/design-system/rollup.config.js:49]: Make SVGs cross-platformed

File: packages/design-system/.storybook/main.js [hunk 6]
```diff
@@ -1,34 +1,27 @@
...
-    //     },
-    //   },
-    // },
   ],
-  framework: "@storybook/react",
+  framework: '@storybook/react',
   webpackFinal: getConfig,
-};
+  core: {
+    builder: 'webpack5',
```
Comment (teimurjan) [inline packages/design-system/.storybook/main.js:25]: Here is the change. The storybook didn't work well with `react-native-web` on Webpack V4. I have changed it to V5.

File: packages/design-system/.storybook/webpack.js [hunk 7]
```diff
@@ -1,5 +1,3 @@
-const { addWebpackAlias, override } = require('customize-cra')
```
Comment (teimurjan) [inline packages/design-system/.storybook/webpack.js:1]: No more libs for Webpack config.

File: packages/design-system/.storybook/webpack.js [hunk 8]
```diff
@@ -22,13 +20,56 @@ const replaceSvgLoader = (config) => {
...
+
+const pipe =
+  (...functions) =>
+  (target) =>
+    functions.reduce((acc, fn) => fn(acc), target)
+
+const getConfig = (config) => {
+  console.log(config.module.rules[3].use[0].options.presets)
+  return pipe(
+    includeNodeModule('react-native-toast-message'),
```
Comment (teimurjan) [inline packages/design-system/.storybook/webpack.js:68]: `react-native-toast-message` is not transpiled by default. We have to force Webpack to do it.

File: packages/design-system/package.json [hunk 9]
```diff
@@ -68,12 +68,12 @@
     "@web3-react/portis-connector": "^6.2.11",
     "@web3-react/walletconnect-connector": "^6.2.10",
     "autoprefixer": "^9",
-    "customize-cra": "^1.0.0",
```
Comment (teimurjan) [inline packages/design-system/package.json:71]: Not used anymore

File: packages/design-system/package.json [hunk 10]
```diff
@@ -21,13 +18,16 @@
     "react": ">=17.0.2",
     "react-dom": ">=17.0.2",
     "react-native": ">=0.66.3",
+    "react-native-safe-area-context": "^3.3.2",
+    "react-native-svg": "^12.1.1",
```
Comment (teimurjan) [inline packages/design-system/package.json:22]: `native-base` peer deps

**Review by diegonzs-roll — APPROVED** [review]
LGTM! I'll create an issue on memberships-ui to use the latest version on of the design system 👍🏻

**Pull request merged**